src/index.css

```diff
@@ -1,5 +1,6 @@
 body {
   margin: 0;
+  padding-left: 15px;
```

Comment:
15? let's give it 16 :D

Reply:
I swapped it for 1rem. Is it any better?